### ISSUE TITLE
[Bug] Not correctly recognized variable with not enough whitespace

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#464]
```
*** Variables ***
${var} 1
```
(1 space after ${var}). Produces:
```
0309 Section variable name should be uppercase (section-variable-not-uppercase)
0401 Robot Framework syntax error: Invalid variable name '${var} 2'. (parsing-error)
0912 Use built-in variable ${EMPTY} instead of leaving variable without value or using backslash (empty-variable)
```
It should not warn on empty-variable, and parsing-error should be probably catched and issued as "not enough whitespace after variable name" or similar rule.
